### ISSUE TITLE
Alert links system container -a11y

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#43708]
the changes made in #43102 are completely wrong and actually breaks basic bootstrap functionality. Instead of writing a fix for the joomla-alert component the changes impact every div that uses the bootstrap alert classes.

they should be reverted so we dont break regular usage of bootstrap alert divs that are used throughout joomla but now have links and buttons that are unreadable thus breaking usability and accessibility

it should look like this
![image](https://github.com/joomla/joomla-cms/assets/1296369/0db89c45-99c8-47f6-ab9f-8dfaa6cd73a7)

but instead the code in #43102 has broken it so it looks like
![image](https://github.com/joomla/joomla-cms/assets/1296369/2667d4f7-57bd-4314-8742-a3554c2da417)

**Comment by cyrezdev** [comment]
I agree, Bootstrap is broken in Atum admin template...

Even if this improves accessibility use : it's better when a link does not look like a button. :-D

`<button class="btn btn-info"> Button </button>`

`<a href="#" class="btn btn-info"> Link styled like a button </a>`

![Image](https://github.com/user-attachments/assets/1dcc3916-b5e3-4d7b-9cd5-27481f7036fd)
